apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ts

```diff
@@ -37,13 +40,13 @@ router.get(
     const enrollmentCounts = await sqldb.queryRows(
       sql.select_enrollment_counts,
       { course_id: res.locals.course.id },
-      z.object({ course_instance_id: CourseInstanceSchema.shape.id, number: z.string() }),
+      z.object({ course_instance_id: CourseInstanceSchema.shape.id, number: z.number() }),
     );
-    res.locals.course_instances.forEach((ci) => {
+    courseInstances.forEach((ci) => {
       const row = enrollmentCounts.find((row) => idsEqual(row.course_instance_id, ci.id));
       ci.number = row?.number || 0;
```

Comment:
"number" is extremely vague. As long as we're changing things here, can we call this `enrollment_count` instead?

Reply:
Updated to `enrollment_count` in [4708ba6](https://github.com/PrairieLearn/PrairieLearn/pull/9812/commits/4708ba69e4b442865ec896dec7a5155fab7d73b4).